snippets/contract-format.js

```diff
@@ -0,0 +1,58 @@
+// WARNING: THE CODE IN THIS FILE WILL BE SHOWN ON THE AGORIC
+// DOCUMENTATION SITE.
+
+// #region contractFormat
+// @ts-check
+// Checks the types as defined in JSDoc comments
+
+// Add imports here
+
+// Optional: you may wish to use the Zoe helpers in /contractSupport
```

Comment:
starting a path with slash means the wrong thing. I use triple-dot.
```suggestion
// Optional: you may wish to use the Zoe helpers in .../contractSupport
```

Reply:
How about the full path? @agoric/zoe/src/contractSupport